console/console.properties

```diff
@@ -233,7 +234,7 @@
 #subject.new.account.notification=[${instanceName}] New account created
 
 # Encoding of the email templates
-# This "�" char should display nicely in a ISO 8859-1 configured editor
+# This "é" char should display nicely in a ISO 8859-1 configured editor
```

Comment:
This change probably broke file encoding.

It should be edited in a latin-1 configured editor. 
Github editor converts it to UTF-8, as stated in the notice.

Reply:
My bad, we do not mention github editor breaks file encoding.